ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineForkchoiceUpdated.java

```diff
@@ -413,9 +412,9 @@ private void logAtInfo(
     LOG.info(
         logMessage,
         status.name(),
-        forkChoice.getHeadBlockHash(),
-        forkChoice.getFinalizedBlockHash(),
-        forkChoice.getSafeBlockHash());
+        AbstractEngineNewPayload.shortenString(forkChoice.getHeadBlockHash().toHexString()),
```

Comment:
same comment about making etherscan debugging more difficult, less important for FCU though I think since can cross ref with Imported log

Reply:
The same as for newPayload call.